### DISCUSSION TITLE
希望能加入肉鸽刷投资判断路线的功能

### DISCUSSION THREAD

**Discussion by malekfmaster** [MaaAssistantArknights/MaaAssistantArknights#5800] [Q&A]
有不期而遇优先不期而遇，傀影的第一层一般只需战斗一次，然后不期而遇一路到诡异行商投资，现在maa强制性默认选上路，致使发生多次战斗，甚至主动挑战紧急不过的情况，严重降低刷投资的效率。

**Comment by fangao-now** [comment]
哥们说的没错~

不过刷投资这玩意，效率再怎么低，挂2,3晚上也差不多要满了，就不用太苛求这么一丁点儿效率了，你说是不是 XD

**Comment by qingdeng001** [comment]
说的没错不过呢投资这玩意你没事干的时候挂着刷几下就满了
